boxen/platforms/factory.go

```diff
@@ -66,6 +73,38 @@ func GetPlatformEmptyStruct(pT string) (Platform, error) {
 	)
 }
 
+// GetPlatformScrapliDefinition sets the scrapli platform definition to a value
+// of the BOXEN_SCRAPLI_PLATFORM_DEFINITION env var or to a default string value.
+func GetPlatformScrapliDefinition(p string) string {
+	scrapliPlatform := os.Getenv("BOXEN_SCRAPLI_PLATFORM_DEFINITION")
+	if scrapliPlatform != "" {
+		return scrapliPlatform
+	}
+
+	// retrieve default scrapli platform url/name
+	// when env var is not set
+	switch p {
+	case PlatformTypeAristaVeos:
+		return AristaVeosScrapliPlatform
+	case PlatformTypeCiscoCsr1000v:
+		return CiscoCsr1000vScrapliPlatform
+	case PlatformTypeCiscoXrv9k:
+		return CiscoXrv9kScrapliPlatform
+	case PlatformTypeCiscoN9kv:
+		return CiscoN9kvScrapliPlatform
+	case PlatformTypeJuniperVsrx:
+		return JuniperVsrxScrapliPlatform
+	case PlatformTypePaloAltoPanos:
+		return PaloAltoPanosScrapliPlatform
+	case PlatformTypeIPInfusionOcNOS:
+		return IPInfusionOcNOSScrapliPlatform
+	case PlatformTypeCheckpointCloudguard:
+		return CheckpointCloudguardScrapliPlatform
+	}
+
+	return ""
+}
+
```

Comment:
@carlmontanari 
to account for user provided scrapli platform definition locations I added this func that returns consts for scrapli core platforms, if no `BOXEN_SCRAPLI_PLATFORM_DEFINITION` is set.

WDYT?

Reply:
lgtm